datashare-app/src/main/java/org/icij/datashare/tasks/BatchDownloadLoop.java

```diff
@@ -19,7 +19,8 @@
 import static org.icij.datashare.tasks.BatchDownloadRunner.NULL_BATCH_DOWNLOAD;
 
 public class BatchDownloadLoop {
-    private final Path DOWNLOAD_DIR = Paths.get(System.getProperty("user.dir")).resolve("app/tmp");
+    private final String DEFAULT_DIR = Paths.get(System.getProperty("user.dir")).resolve("app/tmp").toString();
```

Comment:
why String instead of Path?

Reply:
To be compatible with `propertiesProvider.get` which is an `Optional<String>` used in the constructor below